source/sync/go-to-production/optimize-sync-atlas-usage.txt

```diff
@@ -309,7 +281,7 @@ Summary
 -------
 
 - Device Sync uses space in your synced Atlas cluster to store change history.
-- Backend compaction reduces the space usage of this history.
-- Advanced backend compaction reduces the space usage of this history even more, but can cause client resets for clients who have not connected to the backend in more than the client maximum offline time (in days).
-- Apps that do not specify a client maximum offline time do not apply advanced backend compaction, so clients of any age can sync without experiencing a client reset.
+- Backend compaction reduces the space usage of this history for Partition Based Sync apps.
+- Trimming reduces the space usage for Flexible Sync apps, but can cause client resets for clients who have not connected to the backend in more than the client maximum offline time (in days).
+- Flexible Sync apps that do not specify a client maximum offline time do not apply trimming, so clients of any age can sync without experiencing a client reset.
```

Comment:
When you enable Flexible Sync in an app, client maximum offline time is enabled by default with a value of 30 days. So I don't think it's accurate to say "Flexible Sync apps that do not specify a client maximum offline time" because you don't _have_ to specify this in order for trimming to occur - it's enabled by default. It would probably be more accurate to say "Flexible Sync apps that have disabled client maximum offline time" because that's an additional action you must take to change the default state.
```suggestion
- Flexible Sync apps that have disabled client maximum offline time do not apply trimming, so clients of any age can sync without experiencing a client reset.
```